dev/benchmarking/examples/benchmark_example.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "benchmark_name": "Benchmark 1",
+    "parameter_names": "{arg0,arg1,arg2}",
```

Comment:
I would expect to be able to pass parameter values as well? At least as an opaque string.

Reply:
Yes..It is currently a row of type "jsonb" in the "benchmark_run" table, like this:
https://github.com/apache/arrow/blob/ad1da27ed1e8e7647e7f7d89ec01df6c790e6616/dev/benchmarking/examples/benchmark_run_example.json#L3-L8

However I have not attempted error-checking to confirm that the "parameter_names" entry in the benchmark definition match the keys for "parameter_values" in the benchmark_run.

Reply:
Oh, right, thank you. I was mixing up the benchmark file and benchmark run file. No need to error-check parameters, IMHO.